<!--
%\VignetteEngine{knitr::knitr}
%\VignetteIndexEntry{Salmon quantifications for human macrophage immune response experiment}
-->

# Experiment and sample description

This package provides the output of running Salmon on a set of 24
RNA-seq samples from Alasoo, et al. "Shared genetic effects on
chromatin and gene expression indicate a role for enhancer priming in
immune response", published in Nature Genetics, January 2018 
[doi: 10.1038/s41588-018-0046-7](https://doi.org/10.1038/s41588-018-0046-7).

6 donors were selected from those that had publicly available reads in
ENA. The selection procress is recorded in the file
`inst/scripts/ena_download.R`, the donors were chosen as all female,
the sample having been received as frozen, and then the top 6 were
chosen based on mean purity as recorded by the study authors.

# Salmon quantification

[Salmon](https://combine-lab.github.io/salmon/)
version 0.12.0 was run using 
[Gencode](https://www.gencodegenes.org/)
human reference transcripts v29, with a 
[snakemake](https://snakemake.readthedocs.io/en/stable/) 
script that can be seen in `inst/scripts/Snakemake` and with 
log output in `inst/scripts/snakemake.log`. 

The quantification output is in the `inst/extdata/quant`
directory. The `quant.sf` files have been gzipped to preserve space,
so they are stored as `quant.sf.gz`. There are 20 Gibbs inferential
replicates for each sample.

# Sample information

The `inst/scripts/ena_downloads.R` R script also shows how the column
data file `coldata.csv` was generated. This file contains the
phenotypic information about the 24 samples:

```{r}
dir <- system.file("extdata", package="macrophage")
coldata <- read.csv(file.path(dir,"coldata.csv"))
coldata <- coldata[,c(1,2,3,5)]
coldata
```

# Acknowledgments 

Thanks to the study authors for posting their data publicly and
clearly labelling their data.

# Session info

```{r}
sessionInfo()
```
